src/app/api/banner.ts

```diff
@@ -0,0 +1,15 @@
+import { baseApi } from "./base";
+import type { Banner } from "@shared";
+
+export const bannerApi = baseApi.injectEndpoints({
+  endpoints: (builder) => ({
+    banners: builder.query<Banner, string>({
```

Comment:
Please rename this method. What it does? It get banners from backend, so it can be getBanners()